### ISSUE TITLE
For use with MATE

### ISSUE THREAD

**Issue by adrianvg** [folixg/pause-on-lock#13]
Hi,

I added this bit to your script to make it work with Mate DE as well.
Tested on a HP Elitebook 850, running Ubuntu 18.04 LTS and Mate 1.20.1.
Worked fine, both when locking and unlocking with Spotify Premium, which is my main concern.
Maybe consider supporting Mate as well, unless there are other considerations when Mate is involved?

```
elif [ "$XDG_CURRENT_DESKTOP" == "MATE" ] ; then
  DBUS_LOCK="interface='org.mate.ScreenSaver',member='ActiveChanged'"
  LOCK_SIGNAL="true"
  UNLOCK_SIGNAL="false"
```

**Comment by folixg** [comment]
Thanks for the suggestion. Will include it when I have some spare time.
Of course you can also provide a PR.

**Comment by adrianvg** [comment]
I'm a newbie on Github.
When you say "provide a PR", what do you mean?

Ah, a pull request?
https://help.github.com/en/articles/about-pull-requests

I'll see what I can do!

**Comment by adrianvg** [comment]
Did the PR as requested. I think it went through.

**Comment by folixg** [comment]
Closed with #14.
